src/tests/class_tests/openms/source/File_test.cpp

```diff
@@ -93,13 +93,15 @@ START_SECTION((static String find(const String &filename, StringList directories
   TEST_EXCEPTION(Exception::FileNotFound, File::find(""))
 END_SECTION
 
+#ifdef ENABLE_DOCS
 START_SECTION((static String findDoc(const String& filename)))
   TEST_EXCEPTION(Exception::FileNotFound,File::findDoc("non-existing-documentation"))
   // should exist in every valid source tree (we cannot test for Doxyfile since doxygen might not be installed)
   TEST_EQUAL(File::findDoc("doxygen/Doxyfile.in").hasSuffix("Doxyfile.in"), true)
```

Comment:
Hmm but this only looks for the file in the source tree, which should be fine, right?
If we start making our tests work from an install location this will be a whole new can of worms.